### ISSUE TITLE
FAQ navigation

### ISSUE THREAD

**Issue by LMElston** [gc-proto/covid-19-guidance#97]
We received an email from LPRAB asking to add a table of contents to the FAQ page that includes each question as either a link or a text mention in the table of contents.

I have rejected this idea (though you can champion it if you really want to) but said I would check in to see if we had any other reasonably easy solutions to help people find what they are looking for on this huge page.

Recognizing that:
- obviously this is part of the reason FAQs should not be used in the first place (they know)
- this content is probably only going to be live for 4-5 more months but will continue to grow throughout that time
- I suspect people are already using ctrl-F to navigate this page?
- as Chad mentioned, we don't want to spend a lot of effort designing FAQs solutions and accidentally encouraging their use

@cfarquharson @Christopher-O @Alireme @tanyafowler @lbelmore-gc : Before I respond with a final answer to LPRAB, do any of you have anything you'd like to suggest?

**Comment by LMElston** [comment]
[CEWS - updated Table of Contents.docx](https://github.com/gc-proto/covid-19-guidance/files/5204431/CEWS.-.updated.Table.of.Contents.docx)

(Adding their suggested ToC for reference.)

**Comment by Christopher-O** [comment]
@LMElston in your mock up, would these titles be linked or just text?

**Comment by LMElston** [comment]
That's LPRAB's mock-up, and I don't personally think we should do it (just leave the ToC as-is unless there's another alternative besides having 6+ pages of Qs listed in the ToC). 

For what it's worth, they suggested either of those options though--links or just text.

**Comment by Christopher-O** [comment]
@LMElston I agree.  I ask because if they were just text, there would be accessibility issues for visually impaired users, so we wouldn't impliment that just on that.  If they were links, then I don't see any advantage as I believe most users would still use Cntl+F to navigate through the blue and underline text...which then defeats the purpose, and in fact puts more burden on the user.  I agree that any enhanchements only further pushes the use over the regular content (which should never be the case).  On top, there is so much of the content that is already applied in the regular content, which adds to the FAQ content that should be removed anyways and thus would slim it down.  We should not be aiding in any UX/UI enhanchements anyways unless the content is allowed to be also reformatted.

**Comment by LMElston** [comment]
Just adding a short update to say that this ball is still in LPRAB's court for the time being (they have not decided what they want to do about the proposed ToC).
